src/main/java/picard/arrays/illumina/InfiniumVcfFields.java

```diff
@@ -112,7 +112,8 @@
     public static final String FAIL_REF = "FAIL_REF";
     public static final String ZEROED_OUT_ASSAY = "ZEROED_OUT_ASSAY";
 
-    public enum GENOTYPE_VALUES {AA, AB, BB}
+    public enum GENOTYPE_VALUES {AA, AB, BB};
```

Comment:
see in picard.fingerprint.HaplotypeProbabilities.Genotype how I avoided using the implicit ordinal when referencing array positions.

Reply:
done.